todo_app/app.py

```diff
@@ -1,15 +1,35 @@
-from flask import Flask
-
+from flask import Flask, render_template, request, redirect, url_for
 from todo_app.flask_config import Config
 
+
+from .data import session_items as session
+
 app = Flask(__name__)
 app.config.from_object(Config)
 
 
 @app.route('/')
 def index():
-    return 'Hello World!'
+    #return 'Hello World!'
+    print(session.get_items())
```

Comment:
You normally don't want to keep print statements around even if it was originally helpful for debugging.